gcp_variant_transforms/libs/annotation/vep/vep_runner_test.py

```diff
@@ -103,19 +112,19 @@ def test_make_vep_cache_path(self):
     test_instance = self._create_test_instance()
     self.assertEqual(test_instance._vep_cache_path, _CACHE)
     test_instance = vep_runner.VepRunner(
-        self._mock_service, _SPECIES, _ASSEMBLY, _INPUT_PATTERN, _OUTPUT_DIR,
-        _VEP_INFO_FIELD, _IMAGE, '', _NUM_FORK, self._get_pipeline_args(),
-        None, 30)
+        self._mock_service, _LOCATION, _SPECIES, _ASSEMBLY, _INPUT_PATTERN,
+        _OUTPUT_DIR, _VEP_INFO_FIELD, _IMAGE, '', _NUM_FORK,
+        self._get_pipeline_args(), None, 30)
     self.assertEqual(test_instance._vep_cache_path,
                      ('gs://cloud-lifesciences/vep/'
-                      'vep_cache_homo_sapiens_GRCh38_91.tar.gz'))
+                      'vep_cache_homo_sapiens_GRCh38_104.tar.gz'))
     test_instance = vep_runner.VepRunner(
-        self._mock_service, 'mouse', 'mm9', _INPUT_PATTERN, _OUTPUT_DIR,
-        _VEP_INFO_FIELD, _IMAGE, '', _NUM_FORK, self._get_pipeline_args(),
-        None, 30)
+        self._mock_service, _LOCATION, 'mouse', 'mm9', _INPUT_PATTERN,
+        _OUTPUT_DIR, _VEP_INFO_FIELD, _IMAGE, '', _NUM_FORK,
+        self._get_pipeline_args(), None, 30)
     self.assertEqual(test_instance._vep_cache_path,
                      ('gs://cloud-lifesciences/vep/'
-                      'vep_cache_mouse_mm9_91.tar.gz'))
+                      'vep_cache_mouse_mm9_104.tar.gz'))
```

Comment:
@moschetti There does not exist a `mouse_mm9`, though there is a `mus_musculus`:

```Bash
[pgrosu]$ gsutil ls gs://cloud-lifesciences/vep/
gs://cloud-lifesciences/vep/
gs://cloud-lifesciences/vep/vep_cache_homo_sapiens_GRCh37_104.tar.gz
gs://cloud-lifesciences/vep/vep_cache_homo_sapiens_GRCh37_91.tar.gz
gs://cloud-lifesciences/vep/vep_cache_homo_sapiens_GRCh38_104.tar.gz
gs://cloud-lifesciences/vep/vep_cache_homo_sapiens_GRCh38_91.tar.gz
gs://cloud-lifesciences/vep/vep_cache_mus_musculus_GRCm39_104.tar.gz
[pgrosu]$
```

Reply:
Thanks for catching that. Updated in new commit.